fury/ui/elements.py

```diff
@@ -2630,6 +2631,7 @@ def update(self):
         for slot in self.slots[len(values_to_show):]:
             slot.element = None
             slot.set_visibility(False)
+            slot.resize((self.slot_width, 0))
```

Comment:
Why do we need to `resize` this with `(self.slot_width, 0)`??
As there is no `height` it wouldn't be visible anyways??

Reply:
Hello @ganimtron-10 , Yes, this is the part where the slot height is set to zero and the width is kept unchanged.